core/vm/logger.go

```diff
@@ -304,6 +302,11 @@ func (l *StructLogger) CaptureState(pc uint64, op OpCode, gas, cost uint64, scop
 }
 
 func (l *StructLogger) CaptureStateAfter(pc uint64, op OpCode, gas, cost uint64, scope *ScopeContext, rData []byte, depth int, err error) {
+	stack := scope.Stack
+	if op == SLOAD && stack.len() >= 1 {
+		storageValue := stack.data[stack.len()-1].Bytes32()
+		l.inflightSload(storageValue)
```

Comment:
i noticed you set `inflightSload` to nil in pre state tracer.  Why not here?

Reply:
I think we'd better also need to set it to nil here, I guess Omer forgets